pyproject.toml

```
@@ -96,16 +96,19 @@ test = [
     "boltons",
     "scanpy",
     "httpx", # For data downloading
-    "dask[array,distributed]>=2022.09.2,<2024.8.0",
+    "dask[distributed]>=2022.09.2",
     "awkward>=2.3",
     "pyarrow",
     "pytest_memray",
-    "pytest-mock"
+    "pytest-mock",
+    "anndata[dask]"
```

why do we still miss some trailing commas? Time to format our TOML I think.

```suggestion
    "anndata[dask]",
```